securesignals-gma-dev-app/src/main/java/com/uid2/dev/BannerActivity.java

```diff
@@ -101,7 +104,11 @@ private void loadUID2Identity() {
                 refreshFrom,
                 refreshExpires,
                 fromJsonIdentity.getRefreshResponseKey());
-            UID2Manager.getInstance().setIdentity(identity);
+            if (isEnvironmentEUID(getMetadata(getBaseContext()))) {
```

Comment:
No need to use the base context

```suggestion
            if (isEnvironmentEUID(getMetadata(this))) {
```